src/statement/mod.rs

```diff
@@ -105,6 +114,8 @@ impl<'a, 'b, S> Statement<'a, 'b, S, HasResult> {
         self.raii.num_result_cols().into_result(self)
     }
 
+    pub fn describe_col(&self, idx: u16) -> Result<ColumnDescriptor> { self.raii.describe_col(idx).into_result(self) }
```

Comment:
Just a short line of documentation. We should give a hint to the user that indexing starts at 1